### ISSUE TITLE
NTR: Brain imaging abnormality

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4148]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Brain imaging abnormality

### 2. Synonyms
Abnormal brain imaging

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)
An anomaly of metabolism or structure of the brain identified by imaging.

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
HP_0012638  'Abnormality of nervous system physiology'

This is for the Kid's First data portal.
